src/loki/height_action.cc

```diff
@@ -71,7 +71,7 @@ std::vector<PointLL> loki_worker_t::init_height(Api& request) {
 */
 std::string loki_worker_t::height(Api& request) {
   // time this whole method and save that statistic
-  auto _ = measure_scope_time(request, "loki_worker_t::height");
+  auto _ = measure_scope_time(request);
```

Comment:
there will be a lot of changes that look just like this. i thought it was stupid to have to have the caller know the action and the worker. instead i can figure this out programatically from within the `measure_scope_time` function